main.c

```diff
@@ -304,6 +308,15 @@ STATIC void print_code_py_status_message(safe_mode_t safe_mode) {
     }
 }
 
+#ifdef CIRCUITPY_BOOT_COUNTER
+nvm_bytearray_obj_t bootcnt = {
+    .base = {.type = &nvm_bytearray_type},
+    .len = ( uint32_t) 1,
+    .start_address = (uint8_t*) (0x00080000 - CIRCUITPY_INTERNAL_NVM_SIZE)
+    };
```

Comment:
Why do you need this? Can't you use the nvm object instead?

Reply:
I was never able to get `common_hal_nvm_bytearray_get_bytes(&common_hal_mcu_nvm_obj,0,1,&value_out);` to work since `common_hal_mcu_nvm_obj` is declared as a const. Any thoughts?

Reply:
I think you can add the `const` to the common_hal_nvm_bytearray functions and then it will.